specification/dynamicstelemetry/resource-manager/Microsoft.DynamicsTelemetry/preview/2019-01-24/dynamicstelemetry.json

```diff
@@ -0,0 +1,139 @@
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "info": {
+    "title": "Dynamics Telemetry",
+    "version": "2019-01-24"
+  },
+  "host": "telemetry.dynamics.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.DynamicsTelemetry/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": true,
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Operations": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OperationList": {
+      "type": "array",
+      "description": "List of supported operations.",
+      "items": {
+        "$ref": "#/definitions/Operation"
+      }
+    },
+    "Operation": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "isDataAction"
+      ],
+      "properties": {
+        "origin": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/Display"
+        },
+        "properties": {
+          "$ref": "#/definitions/Properties"
+        },
+        "isDataAction": {
+          "type": "boolean"
+        }
+      }
+    },
+    "Display": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "provider": {
+          "type": "string"
+        },
+        "resource": {
+          "type": "string"
+        },
+        "operation": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "publisher": {
+          "type": "string"
+        }
+      }
+    },
+    "Properties": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "serviceSpecification": {
+          "$ref": "#/definitions/ServiceSpecification"
+        }
+      }
+    },
+    "ServiceSpecification": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "logSpecifications": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LogSpecification"
+          }
+        }
+      }
+    },
+    "LogSpecification": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "displayName": {
+          "type": "string"
+        },
+        "blobDuration": {
+          "type": "string"
```

Comment:
Should this be a string or is it a time duration? If it's a time duration, please follow the standard for ISO8601 times or durations.

Reply:
Hello, "Autorest" plugin for VS Code does not allow to use time-span.
![image](https://user-images.githubusercontent.com/43667256/56138128-e69c8680-5f9e-11e9-9ecb-3893bb4c8170.png)
![image](https://user-images.githubusercontent.com/43667256/56138150-f2884880-5f9e-11e9-80df-223513310c54.png)

Reply:
This should be `format: "date-time"` and include an RFC 3339, section 5.6 formatted string with only millisecond precision.

Reply:
I've checked other teams reposes for example 
\azure-rest-api-specs\specification\securityinsights\resource-manager\Microsoft.SecurityInsights\preview\2019-01-01-preview\SecurityInsights.json

and found that they are using  

I think it is a more suitable case for me.
![image](https://user-images.githubusercontent.com/43667256/56666172-0ffe9600-66b4-11e9-8f08-164bfd79ea2a.png)

![image](https://user-images.githubusercontent.com/43667256/56666109-f9583f00-66b3-11e9-936e-07d7c3783f46.png)

So I've changed the code according to that information.
Do you think it is a right decision?

Reply:
That is the right decision. I didn't read closely enough. Great catch!